cmd/snapweb/main.go

```diff
@@ -29,10 +29,8 @@ import (
 
 var logger *log.Logger
 
-const (
-	httpAddr  string = ":4200"
-	httpsAddr string = ":4201"
-)
+var httpAddr string  // set from go build ldflags
+var httpsAddr string // set from go build ldflags
```

Comment:
What I've done is default the ports to 4200/4201 unless otherwise specified at build.